Makefile

```diff
@@ -2,79 +2,106 @@ include common.mk
 
 BUILDDIR := build
 CLIENTBUILD := $(BUILDDIR)/client
-SERVERBUILD := $(BUILDDIR)/server
-LOCALSERVERBUILD := $(BUILDDIR)/local_server
+SERVERBUILD := $(BUILDDIR)/backend/czi_hosted
```

Comment:
I think it would be clearer if the variables reflected their content, eg, `s/SERVERBUILD/CZIHOSTEDBUILD/g` and `s/LOCALSERVERBUILD/SERVERBUILD/g`.

At your discretion.  But it caused a small bit of confusion for me.

Reply:
updated